src/main/java/com/orasi/bluesource/Accounts.java

```diff
@@ -376,5 +396,137 @@ public void selectCell(int row, int column) {
 		PageLoaded.isDomComplete(driver, 1);
 	}
 	
+	/**
+	 * Clicks on New Project, fills out required fields and submits form
+	 * 
+	 * @author Christopher Batts
+	 */
+	public void addProject() {
+		btnNewProject.click();
+		txtProjectName.sendKeys("New Project 1");
+		txtProjectStart.sendKeys("04062018");
+		txtProjectEnd.sendKeys("06042018");
+		btnCreateProject.click();
+	}
+	
+	/**
+	 * Clicks a project link in the projects table under an account.
+	 * 
+	 * @param String Name of project to select
+	 * @author Christopher Batts
+	 */
+	public void selectProject(String projName) {
+		PageLoaded.isDomComplete(driver, 5);
+		tblProjects.findElement(By.linkText(projName)).syncVisible(5);
+		tblProjects.findElement(By.linkText(projName)).click();
+	}
+	
+	/**
+	 * Clicks the New Role button under a project
+	 * 
+	 * @author Christopher Batts
+	 */
+	public void clickNewRole() {
+		btnNewRole.syncVisible(5);
+		btnNewRole.click();
+	}
+	
+	/**
+	 * Clicks the Billable/Non-Billable toggle
+	 * 
+	 * @author Christopher Batts
+	 */
+	public void toggleBill() {
+		btnBill.syncVisible(5);
+		btnBill.click();
+	}
+	
+	/**
+	 * Checks state of Billable/Non-Billable toggle. Returns true if Billable is displayed, otherwise returns false.
+	 * 
+	 * @return Boolean 
+	 * @author Christopher Batts
+	 */
+	public boolean isBillOn() {
+		return !(btnBill.getAttribute("class").contains("off"));
+	}
+	
+	/**
+	 * Clicks the +New Role button, fills out required fields, and submits the form.
+	 * 
+	 * @author Christopher Batts
+	 */
+	public void createRole() {
+		clickNewRole();
+		if(isBillOn() == true) {
+			toggleBill();
+		}
+		lstRoleType.syncVisible(5);
+		lstRoleType.select("Project Manager");
+		txtMaxResources.sendKeys("25");
+		btnCreateRole.click();
+	}
+	
+	/**
+	 * Selects a role by link text from the Project Roles table
+	 * 
+	 * @param String Role name to be selected
+	 * @author Christopher Batts
+	 */
+	public void selectNewRole(String roleName) {
+		tblProjectRoles.findElement(By.linkText(roleName)).click();
+	}
+	
+	/**
+	 * Fills out employee assignment form and submits
+	 * 
+	 * @author Christopher Batts
+	 */
+	public void assignEmployeeToProject() {
+		lstAssignEmployee.syncVisible(5);
+		lstAssignEmployee.select("test 123");
+		btnCreateFilledRole.click();
+	}
+	
+	public boolean verifyNewProjectIsDisplayed(String projectName) {
+		boolean x = false;
+		try {
+			x = elmQuickNavProjectList.findElement(By.linkText(projectName)).isDisplayed();
+		} catch (Exception e){
+			return x;
+		}
+		return x;
+		
```

Comment:
so you'll want to use the sync methods here.  they all allow a parameter that lets you chose whether to swallow the exception so test can assert on boolean, or to throw it.  
so instead you would do:

`return elmQuickNavProjectList.findElement(By.linkText(projectName)).syncVisible(5, false);`